doc/7/protocols/websocket/constructor/index.md

```diff
@@ -30,8 +30,9 @@ WebSocket protocol connection options.
 | Property            | Type<br/>(default)               | Description                                                                                  |
 | ------------------- | -------------------------------- | -------------------------------------------------------------------------------------------- |
 | `autoReconnect`     | <pre>boolean</pre><br/>(`true`)  | Automatically reconnect to kuzzle after a `disconnected` event                               |
-| `port`              | <pre>number</pre><br/>(`7512`)   | Kuzzle server port                                                                           |
 | `headers`           | <pre>object</pre>(`{}`)          | Connection custom HTTP headers (e.g. origin, subprotocols, ...)<br/>**(Not supported by browsers)** |
+| `pingInterval` | <pre>number</pre><br/>(`10000`)   | Number of milliseconds between two pings                                         |
```

Comment:
As I said, 10 seconds seems to be a lot. In 10 seconds an user can do many interactions with a mobile or web app before the application realize that there is no longer connection with the backend and switch to offline mode.

2 seconds seems to be small enough to have more reactivity.

(@scottinet @Njuelle any thought on this?)

Reply:
I agree, 10 seconds in an app that is currently in use is too long.
2 seconds is ok for me too